src/app/components/RenderMessageContent.tsx

```diff
@@ -67,38 +68,63 @@ export function RenderMessageContent({
       </UrlPreviewHolder>
     );
   };
-
-  const renderFile = () => (
-    <MFile
-      content={getContent()}
-      renderFileContent={({ body, mimeType, info, encInfo, url }) => (
-        <FileContent
-          body={body}
-          mimeType={mimeType}
-          renderAsPdfFile={() => (
-            <ReadPdfFile
-              body={body}
-              mimeType={mimeType}
-              url={url}
-              encInfo={encInfo}
-              renderViewer={(p) => <PdfViewer {...p} />}
+  const content: IImageContent = getContent();
+  const renderCaption = () => {
+    if(content.filename && content.filename !== content.body) {
```

Comment:
Looks like `const content: IImageContent...` left to be pulled inside `renderCaption` function?